src/main/java/net/sf/rails/game/special/SpecialMultiTileLay.java

```diff
@@ -0,0 +1,236 @@
+package net.sf.rails.game.special;
+
+import net.sf.rails.common.LocalText;
+import net.sf.rails.common.parser.ConfigurationException;
+import net.sf.rails.common.parser.Tag;
+import net.sf.rails.game.*;
+import net.sf.rails.game.state.IntegerState;
+import net.sf.rails.util.Util;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author Martin Brumm <dr.martin.brumm@t-online.de>
+ * @date May 20th 2020
+ * This Class is based on the assumption that while you can lay mutiple Tiles, each specific tile can only
+ * be placed once on the map with this SpecialPower for the private with that special power.
+ */
+public class SpecialMultiTileLay extends SpecialTileLay {
+	
+	private String locationCodes = null;
+	private List<MapHex> locations = null;
+	private String name = null;
```

Comment:
`name`, `locations`, `locationCodes`, `extra`, `free`, `discount` and `connected` and `tileColours` all exist in the base class? Assuming we don't really need to declare these here? We can then also ditch the related getters and setters